web-wallet/src/lib/components/Send/Send.svelte

```diff
@@ -120,10 +133,67 @@
         href: "/dashboard",
         isAnchor: true,
       }}
-      nextButton={{ disabled: isNextButtonDisabled }}
+      nextButton={{
+        action: () => {
+          activeStep = 1;
+        },
+        disabled: !addressValidationResult.isValid,
+      }}
     >
       <div in:fade|global class="operation__send">
-        <ContractStatusesList items={statuses} />
+        <div class="operation__send-amount operation__space-between">
+          <p>Enter address:</p>
+          <Button
+            disabled={!scanner}
+            size="small"
+            on:click={() => {
+              scanQrComponent.startScan();
+            }}
+            text="SCAN QR"
+          />
+        </div>
+        <Textbox
+          className="operation__send-address
+          {!addressValidationResult.isValid
+            ? 'operation__send-address--invalid'
+            : ''}"
+          type="multiline"
+          bind:value={address}
+        />
+        <ScanQR
+          bind:this={scanQrComponent}
+          bind:scanner
+          on:scan={(event) => {
+            address = event.detail;
+          }}
+        />
+      </div>
+    </WizardStep>
+    <WizardStep
+      step={1}
+      {key}
+      backButton={{
+        action: () => {
+          activeStep = 0;
+        },
+      }}
+      nextButton={{
+        action: () => {
+          activeStep = 2;
+        },
+        disabled: isNextButtonDisabled,
+      }}
+    >
+      <div in:fade|global class="operation__send">
+        <ContractStatusesList items={statuses}>
+          <AppAnchorButton
+            className="allocate-button"
+            href="/dashboard/allocate"
```

Comment:
I am not sure this is the correct approach here. Allocate should not be a route as we do not have it in the dashboard. This means that the only way to access it is via the `Send` flow. Another problem that this creates is that by accessing the allocate route through the `Shield more Dusk` button, I cannot go back to the first step in the `Send` flow I already completed, but instead I go back to the dashboard. This to me doesn't seem the correct flow.

Reply:
The allocate path is the Shield / Unshield option on the Dashboard.

<img width="623" alt="Screenshot 2024-09-19 at 16 27 18" src="https://github.com/user-attachments/assets/2920e85d-e76b-4377-8c86-090fc2b7b6e6">

Reply:
> Another problem that this creates is that by accessing the allocate route through the Shield more Dusk button, I cannot go back to the first step in the Send flow I already completed, but instead I go back to the dashboard.

I agree with this. It's indeed a poor user experience.

Reply:
![alt text](image.png)
This is what I have in the dashboard. But I think I might be missing a ENV variable.
Still my point stands I would not interupt the flow of the stepper if the user wants to shield dusk. The back button should send me back to where I left not why all my progress. This is my opinion.

Reply:
> ![image](https://private-user-images.githubusercontent.com/28288533/369019416-0bca3875-c3c5-4082-9bf2-f3b93592aa5f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY3NTM3MTYsIm5iZiI6MTcyNjc1MzQxNiwicGF0aCI6Ii8yODI4ODUzMy8zNjkwMTk0MTYtMGJjYTM4NzUtYzNjNS00MDgyLTliZjItZjNiOTM1OTJhYTVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE5VDEzNDMzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJlMGI2YTdkMjM0MGUxMGVlY2E0NGM5MjcyNWViNThjNGEwNWZlZjU2N2MzNGZmNjUyOTRkMTYyYTFhMTMzMWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nU7zAAIYMh-lWfcie7c50mMHQsKzI24w90U6ei_0RnI) This is what I have in the dashboard. But I think I might be missing a ENV variable. Still my point stands I would not interupt the flow of the stepper if the user wants to shield dusk. The back button should send me back to where I left not wipe all my progress. This is my opinion.

Yeah, it's controlled by the VITE_CONTRACT_ALLOCATE_DISABLED var.

But I agree with the rest.

Reply:
For now, this is just the simplest solution.  Inserting the Allocate flow into the Send flow to preserve the flow isn't planned, but can be considered as an improvement later.